zipkin-web/src/main/resources/templates/v2/index.mustache

```diff
@@ -22,7 +22,7 @@
       <label for="timeStamp">End time</label>
       <input class="form-control input-sm date-input" type="text">
       <input class="form-control input-sm time-input" type="text">
-      <input class="" id="endTs" name="endTs" type="hidden" value="{{endTs}}">
+      <input class="timestamp-value" id="endTs" name="endTs" type="hidden" value="{{endTs}}">
```

Comment:
how is the `timeStamp` function scoped? When it does `this.$node.find(".timestamp-value");`, is it guaranteed to only search children of some element, e.g. the div? If it does a global search it could find other timestamp fields with the same class.